BlazorLeaflet/BlazorLeaflet/wwwroot/leafletBlazorInterops.js

```diff
@@ -206,6 +206,20 @@ window.leafletBlazor = {
     },
     getZoom: function (mapId) {
         return maps[mapId].getZoom();
+    },
+    zoomIn: function (mapId, e) {
+        const map = maps[mapId];
+
+        if (map.getZoom() < map.getMaxZoom()) {
+            map.zoomIn(map.options.zoomDelta * (e.shiftKey ? 3 : 1));
+        }
+    },
+    zoomOut: function (mapId, e) {
+        const map = maps[mapId];
+
+        if (map.getZoom() > map.getMinZoom()) {
+            map.zoomOut(map.options.zoomDelta * (e.shiftKey ? 3 : 1));
```

Comment:
It would be probably better if these hard coded values would be configurable by the user.

Reply:
So, we should probably expose the `ZoomDelta` property somewhere (although I'm not sure it belongs in `Map` itself: that class runs risk of getting a _lot_ of properties over time — do we want a `MapSettings` or `MapOptions` class?).

As for the `3`, [Leaflet hardcodes that behavior](https://github.com/Leaflet/Leaflet/blob/37d2fd15ad6518c254fae3e033177e96c48b5012/src/control/Control.Zoom.js#L70). Of course, that doesn't mean we need to do the same. What do we call this? `HighSpeedZoomDelta`, maybe?

Reply:
> do we want a MapSettings or MapOptions class?

I've moved that discussion to a separate issue #31.

If it's fine by you, I would just add `ZoomDelta` and `ZoomDelta_Faster` properties for now. The former would map directly to Leaflet's `map.options.zoomDelta`, but the latter would only have an effect if you run this PR's custom methods, since Leaflet itself hardcodes it to `3`.